tests/test_utils_signing_api.py

```diff
@@ -1,21 +1,21 @@
 from resource_server.utils.signing_api import get_keys, check_signature
 
 # TODO: should verify that get_keys issues correct _post_request
-def test_get_keys(self):
+def test_get_keys(mock_ssh_cert_service):
     """ Get new ssh keys and CA """
-    keys = get_keys()
+    keys = mock_ssh_cert_service.get_keys()
```

Comment:
this is essentially testing mock_ssh_cert_service (it's not touching from `resource_server.utils.signing_api.get_keys` at all), which makes this test and/or this mock  object kinda pointless.